pages/doc/logging_logs_settings.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Customize Logs Settings
+tags: [logs]
+sidebar: doc_sidebar
+permalink: logging_logs_settings.html
+summary: Learn how you can customize logs settings from the Organization Settings page. 
+---
+
+As a user with **Accounts** permission, you can map metrics tags to logs tags, traces tags to logs tags, and customize the time window you see on a chart or Traces Browser when you drill into logs from a chart and trace.
+
+## Map Tags
+
+When you are on a chart, application map, or in the Traces Browser, and drill into the Logs Browser, the tags associated with the metrics or trace are added as search filters to the Logs Browser. If the logs tags are different from the metrics or traces tags, you don't see logs on the Logs Browser. Therefore, it is important that you use the same tags for metrics, traces, and logs.
+
+### Map the Traces Tags to Logs Tags
+
+If your traces and logs use different tags, follow these steps to create a map between the tags:
+
+1. From the gear icon <i class="fa fa-cog"></i> on the toolbar, select **Organization Settings**.
+1. Click **+ Map Traces Tags to Log Tags**.
```

Comment:
Not sure whether we need the plus sign here. Same applies to the other procedure.